client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -0,0 +1,388 @@
+          >
+            Settings
+          </button>
+        </li>
+        <li>
+          <button onClick={() => dispatch(logoutUser())}>Log Out</button>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
+const MoreMenu = () => {
+  // TODO: use selectRootFile selector
+  const rootFile = useSelector(
+    (state) => state.files.filter((file) => file.name === 'root')[0]
+  );
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { newSketch, saveSketch } = useSketchActions();
+  const history = useHistory();
+  const navigate = (url) => history.push(url);
+
+  const cmRef = useContext(CmControllerContext);
+
+  return (
+    <div>
+      <IconButton icon={MoreIcon} />
+      <ul>
+        <b>{t('Nav.File.Title')}</b>
+        <li>
+          <button onClick={newSketch}>{t('Nav.File.New')}</button>
+        </li>
+        <li>
+          <button onClick={() => saveSketch(cmRef.current)}>
+            {t('Common.Save')}
+          </button>
+        </li>
+        <li>
+          <button
+            onClick={() => {
+              navigate(`/p5/sketches`);
+            }}
+          >
+            {t('Nav.File.Examples')}
+          </button>
+        </li>
+        <b>{t('Nav.Edit.Title')}</b>
+        <li>
+          <button onClick={cmRef.current?.tidyCode}>
+            {t('Nav.Edit.TidyCode')}
+          </button>
+        </li>
+        <li>
+          <button onClick={cmRef.current?.showFind}>
+            {t('Nav.Edit.Find')}
+          </button>
+        </li>
+        <b>{t('Nav.Sketch.Title')}</b>
+        <li>
+          <button onClick={() => dispatch(newFile(rootFile.id))}>
+            {t('Nav.Sketch.AddFile')}
+          </button>
+        </li>
+        <li>
+          <button onClick={() => dispatch(newFolder(rootFile.id))}>
+            {t('Nav.Sketch.AddFolder')}
+          </button>
+        </li>
+        {/* TODO: Add Translations */}
+        <b>Settings</b>
+        <li>
+          <button
+            onClick={() => {
+              dispatch(openPreferences());
+            }}
+          >
+            Preferences
+          </button>
+        </li>
+        <li>
+          <button>Language</button>
+        </li>
+        <b>{t('Nav.Help.Title')}</b>
+        <li>
+          <button>{t('Nav.Help.KeyboardShortcuts')}</button>
+        </li>
+        <li>
+          <button
+            onClick={() => {
+              window.location = 'https://p5js.org/reference/';
+            }}
+          >
+            {t('Nav.Help.Reference')}
+          </button>
+        </li>
+        <li>
+          <button
+            onClick={() => {
+              navigate(`/about`);
+            }}
+          >
+            {t('Nav.Help.About')}
+          </button>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
+MoreMenu.propTypes = {
+  cmController: PropTypes.shape({
+    tidyCode: PropTypes.func,
+    showFind: PropTypes.func,
+    showReplace: PropTypes.func,
+    getContent: PropTypes.func
+  })
```

Comment:
We can use your context here!  Either now or later.

Reply:
we're already using it, forgot to remove it from here, done!